src/Range/Pattern.php

```diff
@@ -118,6 +118,29 @@ public static function fromString($range, $supportNonDecimalIPv4 = false)
 
             return new static($fromAddress, $toAddress, $asterisksCount);
         }
+        if (strpos($range, '/') !== false) {
```

Comment:
If a string contains `/` it may be a `Range\Subnet` pattern, isn't it?

Reply:
Did I implement this in the wrong class? Sorry, I'll move it over there, then.

Reply:
Thank you for the pointer, the `Range\Subnet` needed only a tiny adjustment. Reverted the changes in `Range\Pattern`.